packages/data-controls/package.json

```
@@ -0,0 +1,31 @@
+{
+	"name": "@wordpress/data-controls",
+	"version": "1.0.0",
```

```suggestion
	"version": "1.0.0-beta.1",
```
See: https://github.com/WordPress/gutenberg/tree/19d1cbfd43daeaf53fc9b3987e2c21eff54aab4b/packages#creating-a-new-package.

Lerna bumps version on every release so we need to start with a version lower than `1.0.0`.